### ISSUE TITLE
Target size minimum (24*24) for the control 'Maximize terminal' present around other control is failed: A11y_Visual Studio Code Client_Integrated Terminal_Maximize terminal_TargetSize

### ISSUE THREAD

**Issue by Swati1700** [microsoft/vscode#210216]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”
**Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.**

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #SH_Visual Studio Code Client_Win32_April2024; #Visual Studio Code Client; #Win32; #Benchmark; #Win11; #WCAG2.5.8; #FTP; #A11yWCAG2.2; #TargetSize;

### Environment and OS details:
Version: 1.89.0-insider (user setup)
Windows 11 pro Version 23H2
OS build 22631.3296

### Repro Steps:
1. Open Visual Studio Code Insiders editor.
2. Using "Ctrl+`(esc)" key or new terminal open the Integrated Terminals.
3. Tab to "Maximize panel size button".
4. Start AI4D tool and inspect.
5. Observe Target size minimum (24*24) for the control 'Maximize terminal' present around other control is failed.

### Actual Result:
Target size minimum (24*24) for the control 'Maximize terminal' present around other control is failed.
For the control 'Maximize terminal', target size is 22*22 which is less than 24*24.
For the control 'Split', target size is 20*20 which is less than 24*24.
For the control 'Kill(Delete)', target size is 20*20 which is less than 24*24.

### **Note: Similar issue is also observed for the below scenarios. **
**Issue: Integrated Terminal - Split terminal** 

1. Open Visual Studio Code Insiders editor.
2. Using "Ctrl+`(esc)" key or new terminal open the Integrated Terminals.
3. Tab to "Split terminal button".
4. Start AI4D tool and inspect.
5. Observe Target size minimum (24*24) for the control 'Split terminal' present around other control is failed.

**Issue: Integrated Terminal - Remove terminal**

1. Open Visual Studio Code Insiders editor.
2. Using "Ctrl+`" key or new terminal open the Integrated Terminals.
3. Tab to "Remove terminal button".
4. Start AI4D tool and inspect.
5. Observe Target size minimum (24*24) for the control 'Remove terminal' present around other control is failed.

### Expected Result:
Target size minimum (24*24) for the control 'Maximize terminal' present around other control should be 24*24.

### User Impact:
Users with mobility or fine motor control impairments will not be able to accessible or select the button properly.

**Comment by meganrogge** [comment]
Closing for same reason as the other 24 by 24 button issue - this button gets larger when a user zooms.
